inc/admin/ui/modules/cdn.php

```diff
@@ -137,7 +141,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'cdn_reject_files',
 			'label_screen' => __( 'Exclude files:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/style.css\ne.g. /wp-content/plugins/some-plugin/(.*).css"
+			'placeholder'  => "/wp-content/themes/some-theme/style.css\n/wp-content/plugins/some-plugin/(.*).css"
```

Comment:
[MDN reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) wasn’t very clear about it, but [spec is](https://www.w3.org/TR/html5/forms.html#the-placeholder-attribute), so changing it.